core/domain/user_services_test.py

```diff
@@ -472,6 +495,38 @@ def test_contribution_msec_does_not_change_if_collection_unpublished(self):
             self.owner_id).first_contribution_msec)
 
 
+class UserDashboardStatsTests(test_utils.GenericTestBase):
+    """Test whether exploration-related statistics of a user change as events
+    are registered.
+    """
+
+    OWNER_EMAIL = 'owner@example.com'
+    OWNER_USERNAME = 'owner'
+    EXP_ID = 'exp1'
+
+    USER_SESSION_ID = 'session1'
+
+    def setUp(self):
+        super(UserDashboardStatsTests, self).setUp()
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+
+    def test_get_user_dashboard_stats(self):
+        exploration = self.save_new_valid_exploration(
+            self.EXP_ID, self.owner_id, end_state_name='End')
+        init_state_name = exploration.init_state_name
+        event_services.StartExplorationEventHandler.record(
+            self.EXP_ID, 1, init_state_name, self.USER_SESSION_ID, {},
+            feconf.PLAY_TYPE_NORMAL)
+        ModifiedUserStatsAggregator.start_computation()
+        self.process_and_flush_pending_tasks()
+        self.assertEquals(
```

Comment:
You should do a "before" check as well by calling the same function before the computation is run.